aviary/interface/methods_for_level2.py

```diff
@@ -1585,9 +1585,9 @@ def add_driver(self, optimizer=None, use_coloring=None, max_iter=50, debug_print
             driver.options["maxiter"] = max_iter
             driver.options["disp"] = True
 
-        if debug_print:
-            if isinstance(debug_print, list):
-                driver.options['debug_print'] = debug_print
+        if verbosity != Verbosity.QUIET:
+            if isinstance(verbosity, list):
+                driver.options['debug_print'] = verbosity
```

Comment:
Because `verbosity` is only ever an enum value, not a list, this logic doesn't hold. There wouldn't be a way to have the user set a custom amountn of output. Here the default verbosity results in `debug_print` being turned on; I'd suggest  it only being on for `VERBOSE` and `DEBUG` if you keep the default as `BRIEF`. Also you can just remove the list logic.